domains/nazmulnahid.json

```
@@ -0,0 +1,18 @@
+{
+  "owner": {
+    "username": "nazmulnahid-git",
+    "email": "nazmulnahid.official@gmail.com"
+  },
+  "records": {
+    "CNAME": "nazmulnahid-git.github.io"
+  },
+  "redirect_config": {
+    "custom_paths": {
+      "/github": "https://github.com/nazmulnahid-git",
+      "/facebook": "https://www.facebook.com/itsnazmulnahid",
+      "/instagram": "https://www.instagram.com/_nazmulnahid",
+      "/linkedin": "https://www.linkedin.com/in/itsnazmulnahid"
+    },
+    "redirect_paths": true
+  }
```

you cant use this redirect_config if you are using CNAME
```suggestion
  }
```